docs/Finnhub API Documentation.md

```diff
@@ -24,7 +20,7 @@ Finnhub does not directly support Java, but we handle API calls using the OkHttp
 **Sample request:**
 
 ```txt
-https://finnhub.io/api/v1/quote?symbol=AAPL&token=YOUR_API_TOKEN
+https://finnhub.io/api/v1/quote?symbol=AAPL&token={YOUR_API_TOKEN}
```

Comment:
This is just a minor comment, usually fields to fill in are indicated by <>. e.g <YOUR_API_TOKEN>. Unless the use of {} here is specific to Markdown  grammar

Reply:
Checked some api docs, it's better to remove the surrounding ones.